generic/tests/nfs_corrupt.py

```diff
@@ -229,15 +265,14 @@ def check_vm_status(vm, status):
     session = vm.wait_for_login(timeout=int(params.get("login_timeout", 360)))
 
     nfs_devname = get_nfs_devname(params, session)
-
     # Write disk on NFS server
     error_context.context("Write disk that image on NFS", logging.info)
     write_disk_cmd = "dd if=/dev/urandom of=%s" % nfs_devname
```

Comment:
I retest it with the following code:
```pythong
    nfs_devname = get_nfs_devname(params, session).strip('\n')
    # Write disk on NFS server
    error_context.context("Write disk that image on NFS", logging.info)
    #write_disk_cmd = "dd if=/dev/urandom of=%s" % nfs_devname
    write_disk_cmd = "dd if=/dev/zero of=%s oflag=direct &" % nfs_devname
    #session.sendline(write_disk_cmd)
    #try:
        # Read some command output, it will timeout
    #    session.read_up_to_prompt(timeout=30)
    #except Exception:
    #    pass
    session.cmd_output(write_disk_cmd)
```

(workspace) [root@ibm-p8-rhevm-01 kar]# python ConfigTest.py --driveformat=virtio_scsi --imageformat=qcow2  --guestname=RHEL.7.6 --platform=ppc64le --mem=8192 --vcpu=8  --testcase=nfs_corrupt  --clone=no 
 (1/1) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.nfs_corrupt: PASS (101.24 s)

Reply:
The qmp log:
2018-10-29 02:38:50: {"execute": "cont", "id": "AJJN0Nww"}
2018-10-29 02:38:50: {"return": {}, "id": "AJJN0Nww"}
2018-10-29 02:38:50: {"execute": "query-status", "id": "TXXiTJaQ"}
2018-10-29 02:38:50: {"return": {"status": "running", "singlestep": false, "running": true}, "id": "TXXiTJaQ"}
2018-10-29 02:38:50: {"execute": "query-status", "id": "GQrN6L0r"}
2018-10-29 02:38:50: {"return": {"status": "running", "singlestep": false, "running": true}, "id": "GQrN6L0r"}
2018-10-29 02:38:50: {"execute": "query-status", "id": "qPaspS8u"}
2018-10-29 02:38:50: {"return": {"status": "running", "singlestep": false, "running": true}, "id": "qPaspS8u"}
2018-10-29 02:38:50: {"execute": "query-status", "id": "HaNxjmNV"}
2018-10-29 02:38:50: {"return": {"status": "running", "singlestep": false, "running": true}, "id": "HaNxjmNV"}
2018-10-29 02:38:51: {"execute": "query-status", "id": "7qq3Ef0N"}
2018-10-29 02:38:51: {"return": {"status": "running", "singlestep": false, "running": true}, "id": "7qq3Ef0N"}
2018-10-29 02:38:51: {"execute": "query-status", "id": "n0TNPtEE"}
2018-10-29 02:38:51: {"return": {"status": "running", "singlestep": false, "running": true}, "id": "n0TNPtEE"}
2018-10-29 02:38:52: {"execute": "query-status", "id": "RmN9ODRs"}
2018-10-29 02:38:52: {"return": {"status": "running", "singlestep": false, "running": true}, "id": "RmN9ODRs"}
2018-10-29 02:38:52: {"execute": "query-status", "id": "U98zEk9O"}
2018-10-29 02:38:52: {"return": {"status": "running", "singlestep": false, "running": true}, "id": "U98zEk9O"}
2018-10-29 02:38:53: {"timestamp": {"seconds": 1540795132, "microseconds": 894629}, "event": "RTC_CHANGE", "data": {"offset": 1}}
2018-10-29 02:38:53: {"execute": "query-status", "id": "208e2n5H"}
2018-10-29 02:38:53: {"return": {"status": "running", "singlestep": false, "running": true}, "id": "208e2n5H"}
2018-10-29 02:38:53: {"execute": "query-status", "id": "g4nmPtEn"}
2018-10-29 02:38:53: {"return": {"status": "running", "singlestep": false, "running": true}, "id": "g4nmPtEn"}
2018-10-29 02:38:54: {"execute": "query-status", "id": "o49hNFAK"}
2018-10-29 02:38:54: {"return": {"status": "running", "singlestep": false, "running": true}, "id": "o49hNFAK"}
2018-10-29 02:38:54: {"execute": "query-status", "id": "J5YwsH3k"}
2018-10-29 02:38:54: {"return": {"status": "running", "singlestep": false, "running": true}, "id": "J5YwsH3k"}
2018-10-29 02:38:55: {"execute": "query-status", "id": "gNLAd9jB"}
2018-10-29 02:38:55: {"return": {"status": "running", "singlestep": false, "running": true}, "id": "gNLAd9jB"}
2018-10-29 02:38:55: {"execute": "query-status", "id": "ZXE9at3L"}
2018-10-29 02:38:55: {"return": {"status": "running", "singlestep": false, "running": true}, "id": "ZXE9at3L"}
2018-10-29 02:38:56: {"execute": "query-status", "id": "YjW4PKtw"}
2018-10-29 02:38:56: {"return": {"status": "running", "singlestep": false, "running": true}, "id": "YjW4PKtw"}
2018-10-29 02:38:56: {"execute": "query-status", "id": "rq7V1v6H"}
2018-10-29 02:38:56: {"return": {"status": "running", "singlestep": false, "running": true}, "id": "rq7V1v6H"}
2018-10-29 02:38:57: {"execute": "query-status", "id": "VWx8vDVP"}
2018-10-29 02:38:57: {"return": {"status": "running", "singlestep": false, "running": true}, "id": "VWx8vDVP"}
2018-10-29 02:38:57: {"execute": "query-status", "id": "4urHPUPZ"}
2018-10-29 02:38:57: {"return": {"status": "running", "singlestep": false, "running": true}, "id": "4urHPUPZ"}
2018-10-29 02:38:58: {"execute": "query-status", "id": "VjekqCWM"}
2018-10-29 02:38:58: {"return": {"status": "running", "singlestep": false, "running": true}, "id": "VjekqCWM"}
2018-10-29 02:38:58: {"execute": "query-status", "id": "2zuhmk4A"}
2018-10-29 02:38:58: {"return": {"status": "running", "singlestep": false, "running": true}, "id": "2zuhmk4A"}
2018-10-29 02:38:59: {"timestamp": {"seconds": 1540795139, "microseconds": 443485}, "event": "BLOCK_IO_ERROR", "data": {"device": "drive_stg", "nospace": false, "__com.redhat_reason": "eio", "node-name": "#block328", "reason": "Input/output error", "operation": "write", "action": "stop"}}
2018-10-29 02:38:59: {"execute": "query-status", "id": "xijZabB2"}
2018-10-29 02:38:59: {"timestamp": {"seconds": 1540795139, "microseconds": 511832}, "event": "STOP"}

Reply:
The above is only my thought :)
@lolyu  @CongLi